client/src/pages/Register.js

```diff
@@ -48,28 +44,18 @@ class Register extends Component {
       confirmPassword: this.state.confirmPassword
     };
     //make http post request to send name,email and password to server
-    // const { data } = await axios.post("/api/users", userData)
-    // console.log("new user info:", data);
-
-    await axios
-      .post("/api/users", userData)
-      .then(response => {
-        console.log("success! Data receieved = ", response.data);
-        //clear input fields
-        this.setState({
-          userName: "",
-          userEmail: "",
-          userPassword: "",
-          confirmPassword: ""
-        });
+    const { data } = await axios.post("/api/users", userData)
+    console.log("Success! Data Received:", data);
+    //clear input fields
+    this.setState({
+      userName: " ",
+      userEmail: " ",
+      userPassword: "",
+      confirmPassword: " "
+    })
 
-        //direct user to profile page
-        this.props.history.replace("/profile");
-      })
-      .catch(error => {
-        console.log("ERROR = ", error.response.data);
-        this.setState({ errorMessage: error.response.data });
```

Comment:
This catch block was how I handled displaying the error message from the server. How come it was removed?